.github/workflows/win_python_matrix_test.yml

```diff
@@ -0,0 +1,85 @@
+name: Windows Pyomo Tests Workflow (Slim, Matrixed, Nightly)
```

Comment:
Similar to #1232, can we change this name to `continuous-integration/github/pr/win`?

Reply:
Has been changed.